internal/api_ui/server.go

```diff
@@ -703,7 +721,11 @@ func (s *Server) CreateLinkQrCodeCallback(ctx context.Context, request CreateLin
 		return CreateLinkQrCodeCallback400JSONResponse{N400JSONResponse{Message: "error getting reverse hash service settings"}}, nil
 	}
 
-	err = s.linkService.IssueClaim(ctx, request.Params.SessionID.String(), s.cfg.APIUI.IssuerDID, *userDID, request.Params.LinkID, s.cfg.APIUI.ServerURL, rhsSettings.CredentialStatusType)
+	if !s.networkResolver.IsCredentialStatusTypeSupported(rhsSettings, credentialStatusType) {
+		return CreateLinkQrCodeCallback400JSONResponse{N400JSONResponse{Message: fmt.Sprintf("Credential Status '%s' is not supported by the issuer", credentialStatusType)}}, nil
+	}
+
+	err = s.linkService.IssueClaim(ctx, request.Params.SessionID.String(), s.cfg.APIUI.IssuerDID, *userDID, request.Params.LinkID, s.cfg.APIUI.ServerURL, credentialStatusType)
 	if err != nil {
 		log.Debug(ctx, "error issuing the claim", "error", err)
```

Comment:
Not for this PR, but this should be a log.Error

Reply:
fixed